templates/artist/artist_detail.html

```diff
@@ -24,26 +24,24 @@ <h1 class="header-font">{{ artist.name }}</h1>
     <div class="break">
         <hr>
     </div>
-    <div class="extra-padding">
         {% thumbnail artist.photo.img "500" as thumb %}
-            <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ artist.name }}" />
+            <img class="extra-padding" src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ artist.name }}" />
         {% endthumbnail %}
-    </div>
     <!-- Genre -->
     {% with genres=artist.genres.all %}
-    <div class="break">
-        <hr>
-    </div>
-    <h4>Genre{{ genres|length|pluralize }}: {{ genres|join:" / " }}</h4>
-    {% endwith %}
-    <h4>Location: {{ artist.location }}</h4>
-    <div class="break">
-        <hr>
-    </div>
+        <div class="break">
+            <hr>
+        </div>
+        <h4>Genre{{ genres|length|pluralize }}: {{ genres|join:" / " }}</h4>
+        {% endwith %}
```

Comment:
Lines 36-40 should be unindented. We only wanted to indent the section inside the `{% with %}` tag.